scripts/reformat-measured-features.js

```diff
@@ -0,0 +1,51 @@
+
+const fsPromises = require('fs').promises;
+
+const features = require('../data/measured-features');
+const FirebaseHandler = require('./firebase-handler');
+
+const firebaseHandler = new FirebaseHandler('v1_1');
+
+const writeCellFeatureData = () => {
+
+    return fsPromises.readFile('./cell-feature-analysis.json')
```

Comment:
parameterize to pass in path to cell-feature-analysis.json and the FirebaseHandler's ctor version string